### PR TITLE
Print hello world

### PR DESCRIPTION
**Pull request by CJ-pdx**
Added a print statement that prints hello and a print statement that prints the date and time.

### CODE REVIEW

File: src/edu/pcc/cis/hello_world/HelloWorld.java [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
+ *
+ * A simple program that prints hello.
+ *
+ */
 public class HelloWorld {
+
+    public static void main (String[] args)
+    {
+        System.out.println("Hello World!");
+        System.out.println(new Date());
```
Comment (CJ-pdx) [inline src/edu/pcc/cis/hello_world/HelloWorld.java]: A comment on line 13

File: src/edu/pcc/cis/hello_world/HelloWorld.java [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+/**
+ * @author Chris Jacobs
+ * @version 05.02.2018
+ *
+ * A simple program that prints hello.
+ *
+ */
 public class HelloWorld {
+
+    public static void main (String[] args)
```
Comment (CJ-pdx) [inline src/edu/pcc/cis/hello_world/HelloWorld.java:13]: A comment on line 13

File: src/edu/pcc/cis/hello_world/HelloWorld.java (hunk 1 above)
Comment (caratang) [inline src/edu/pcc/cis/hello_world/HelloWorld.java:16]: Code looks good.

File: src/edu/pcc/cis/hello_world/HelloWorld.java (hunk 2 above)
Comment (caratang) [reply src/edu/pcc/cis/hello_world/HelloWorld.java:13]: Comment looks good

**Pull request merged**